pkg/extensions/apps_values.go

```diff
@@ -0,0 +1,167 @@
+package extensions
```

Comment:
package "extension"? Is this a leftover from the fact that 'app' were once extensions?